app/views/container_topology/show.html.haml

```diff
@@ -24,56 +25,56 @@
   .legend
     %label#selected
     %div{'ng-if' => "kinds"}
-      %kubernetes-topology-icon{:kind => "ContainerReplicator", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+      %kubernetes-topology-icon{tooltipOptions, :kind => "ContainerReplicator"}
```

Comment:
Oh, interesting.. it seems the splatted hash must only contain symbol keys, not strings. ( `{"abc" => 5, **{:def => 6}}` works but `{"abc" => 5, **{"def" => 6}}` does not)

But converting to strings would be ugly .. so maybe your original solution makes most sense..